api/src/main/java/org/eclipse/microprofile/metrics/annotation/NamedMetadatas.java

```diff
@@ -0,0 +1,36 @@
+/*
+ **********************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICES file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ **********************************************************************/
+
+package org.eclipse.microprofile.metrics.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface NamedMetadatas {
```

Comment:
Is this a reasonable name? I wasn't sure what to pick, the word "metadatas" does sound silly. Should it be `NamedMetadatas` or perhaps `NamedMetadataMulti` or something else?

Reply:
I would go with `NamedMetadatas`.  To be grammatically correct I guess we could rename NamedMetadata to NamedMetadatum... but that sounds even sillier than NamedMetadatas IMO.